### PR TITLE
Ansible systemtests scripts

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

File: templates/install/ansible/playbooks/openshift/systemtests-dependecies.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- hosts: localhost
+  roles:
+    - systemtests_dependencies
```
Comment (lulf) [inline templates/install/ansible/playbooks/openshift/systemtests-dependecies.yml]: Can you just push this into the systemtests.yml playbook?

Comment (kornys) [reply templates/install/ansible/playbooks/openshift/systemtests-dependecies.yml]: Sure that is not a problem.

File: systemtests/scripts/test_func.sh [hunk 2]
```diff
@@ -28,6 +28,10 @@ function setup_test() {
     oc adm --config ${KUBEADM} policy add-cluster-role-to-user cluster-admin $OPENSHIFT_USER
     export OPENSHIFT_TOKEN=`oc whoami -t`
     ansible-playbook ${ENMASSE_DIR}/ansible/playbooks/openshift/systemtests.yml --extra-vars "namespace=${OPENSHIFT_PROJECT}"
+    if [ $? -ne 0 ]; then
+        echo "DEPLOYMENT FAILED - tests won't be executed."
+        exit 1
```
Comment (lulf) [inline systemtests/scripts/test_func.sh]: Maybe out this check into run_test_component? I think other abort conditions are defined there as well

**Comment by kornys** [comment]
@lulf I think I can merge this with master. I removed unused stages from jenkins files (install clients, webdrivers, run Xvfb etc...) and now these steps are executed with your deploy openshift ansible. PR job in Jenkins looks works. Do you agree?

**Comment by kornys** [comment]
@lulf I'm not sure if I understand correctly. You mean remove maven installation steps and trust that on system some maven is already installed? Or try to install maven by yum from rhscl or epel?

**Comment by lulf** [comment]
@kornys To install it from yum

**Comment by kornys** [comment]
@lulf I have to use apache maven > 3.5.0 due to artemis core client, when I try to compile cli-java by maven from yum, If fails.

**Pull request merged**